Makefile

```diff
@@ -365,9 +366,9 @@ lint-go-windows:
 	@GOOS= GOARCH= $(GO) install $(GOLANGCI_LINT_PACKAGE)
 	golangci-lint run
 
-.PHONY: lint-go-vet
-lint-go-vet: ## lint go files with vet
-	@echo "Running go vet..."
+.PHONY: lint-go-gitea-vet
```

Comment:
Looks like it's just `vet`, maybe just name it `go-vet`. The file is Gitea specify, so we don't need `gitea` in the name.

Reply:
No, this task runs _only_ gitea-vet, it runs nothing of `go vet`. The `-vettool` makes it so `gitea-vit` completely replaces all built-in checks.

`go vet` is ran as part of `lint-go` via golangci.